aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -352,6 +354,28 @@ public class AwsProperties implements Serializable {
 
   public static final String HTTP_CLIENT_TYPE_DEFAULT = HTTP_CLIENT_TYPE_URLCONNECTION;
 
+  /**
+   * Used to configure the connection timeout in milliseconds for {@link
+   * software.amazon.awssdk.http.apache.ApacheHttpClient.Builder}. This flag only works when {@link
+   * #HTTP_CLIENT_TYPE} is set to {@link #HTTP_CLIENT_TYPE_APACHE}
+   *
+   * <p>For more details, see
+   * https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/http/apache/ApacheHttpClient.Builder.html
+   */
+  public static final String APACHE_HTTP_CLIENT_CONNECTION_TIMEOUT_MS =
+      "http-client.apache.connection-timeout-ms";
+
+  /**
+   * Used to configure the socket timeout in milliseconds for {@link
+   * software.amazon.awssdk.http.apache.ApacheHttpClient.Builder}. This flag only works when {@link
+   * #HTTP_CLIENT_TYPE} is set to {@link #HTTP_CLIENT_TYPE_APACHE}
+   *
+   * <p>For more details, see
+   * https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/http/apache/ApacheHttpClient.Builder.html
+   */
+  public static final String APACHE_HTTP_CLIENT_SOCKET_TIMEOUT_MS =
```

Comment:
can we match variable name with config name? e.g. `HTTP_CLIENT_APACHE_SOCKET_TIMEOUT_MS`, `httpClientApacheSocketTimeout`

Reply:
Thank you for your suggestion. I've updated them to `HTTP_CLIENT_APACHE_SOCKET_TIMEOUT_MS` and `HTTP_CLIENT_APACHE_CONNECTION_TIMEOUT_MS`